python/demo/demo_scattering_boundary_conditions.py

```diff
@@ -442,7 +442,7 @@ def curl_2d(f: fem.Function):
     )
 
 model = MPI.COMM_WORLD.bcast(model, root=0)
-domain, cell_tags, facet_tags = io.gmshio.model_to_mesh(model, MPI.COMM_WORLD, 0, gdim=2)
+domain, cell_tags, facet_tags, _, _ = io.gmshio.model_to_mesh(model, MPI.COMM_WORLD, 0, gdim=2)
```

Comment:
We should encourage users to used the NamedTuple directly in one of these demos.

Reply:
Fixed in https://github.com/FEniCS/dolfinx/pull/3547/commits/1c78264b4a28a76b442c98632b05c3d7bbb44436